Assets/VRTK/Scripts/Controls/3D/VRTK_Button.cs

```diff
@@ -464,6 +482,11 @@ protected virtual bool ReachedActivationDistance()
             return (Vector3.Distance(transform.position, restingPosition) >= activationDistance);
         }
 
+        protected virtual bool ReachedDeactivationDistance()
+        {
+            return (Vector3.Distance(transform.position, restingPosition) <= Vector3.kEpsilon);
```

Comment:
You can just use `Vector3.==` here since it [returns true if two vectors are approximately equal.](https://docs.unity3d.com/ScriptReference/Vector3-operator_eq.html).